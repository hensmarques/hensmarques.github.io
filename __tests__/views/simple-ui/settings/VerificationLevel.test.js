import React from 'react';
import VerificationLevel from '../../../../src/views/simple-ui/settings/VerificationLevel';

describe('<VerificationLevel/>', () => {
  it('renders and matches the snapshot', () => {
    const wrapper = shallow(<VerificationLevel />);
    expect(wrapper).toMatchSnapshot();
  });
});
