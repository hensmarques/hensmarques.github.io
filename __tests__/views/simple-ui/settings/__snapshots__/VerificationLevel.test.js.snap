// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<VerificationLevel/> renders and matches the snapshot 1`] = `
<SettingsWrapper
  name="Verification Level"
>
  For your security, some of our services require for you to provide basic or additional levels of verification. To verify your account or to check what your current verification level is, please click below.
  <button
    className="login-btn"
  >
    VERIFY ACCOUNT
  </button>
</SettingsWrapper>
`;
